database/migrations/2025_03_12_000000_update_events_table.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        Schema::table('events', function (Blueprint $table) {
+            $table->dropUnique('events_slug_unique');
+            $table->dropColumn('slug');
+        });
+    }
+
+    public function down(): void
+    {
+        Schema::table('events', function (Blueprint $table) {
+            $table->string('slug', 20)->after('image_asset_path');
+            $table->unique('slug', 'events_slug_unique');
```

Comment:
This won't work with existing records in the table. I'm fine with leaving the unique constraint off for the reverse migration step.